cppclient/api_json.cc

```diff
@@ -36,6 +36,14 @@ using iota::model::TryteTransaction;
 DEFINE_uint32(maxNumAddressesForGetBalances, 1000,
               "Maximum number of addresses to query for in 'getBalances'");
 
+DEFINE_uint32(maxQuerySizeFindTransactions, 500,
```

Comment:
this can probably be increased to 5000 or so as well

Reply:
it failed on 2080

Reply:
fair enough, let's keep it this low then.